MapboxVision/VisionManager/VisionReplayManager.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  VisionReplayManager.swift
+//  MapboxVision
+//
+//  Created by Alexander Pristavko on 5/17/19.
+//  Copyright © 2019 Mapbox. All rights reserved.
+//
+
+import Foundation
+import CoreMedia
+
+public final class VisionReplayManager: BaseVisionManager {
+
+    public static func create(recordPath: String) throws -> VisionReplayManager {
+        return VisionReplayManager(dependencies: try ReplayDependencies.default(recordPath: recordPath))
+    }
+
+    public var videoSource: VideoSource {
+        return dependencies.player
+    }
+
+    /**
```

Comment:
Should we add description about thread-safety and expected pattern of sequential calls? Looks like lifecycle methods are not thread-safe.

Reply:
Will add methods about `start`/`stop` sequence.